src/components/Disturber.tsx

```diff
@@ -0,0 +1,170 @@
+
+  // find the closest item to the current date/time
+  const closestItem: DataItem | null = findClosestItem(isActiveData || []);
+
+  const setDistruberComplete = () => {
+    setIsVisiblle((prev) => !prev);
+    addToStore(name, closestItem?.id.toString());
+  };
+
+  useEffect(() => {
+    readFromStore(name).then((value) => {
+      if (closestItem?.id && value !== closestItem?.id.toString()) setIsVisiblle(true);
+    });
+  }, [closestItem]);
+
+  if (!isVisible || !closestItem) return null;
+
+  return (
+    <Wrapper style={{ padding: normalize(24) }}>
+      <View style={[styles.containerRadius, { backgroundColor: closestItem.backgroundColor }]}>
+        {!!closestItem && (
+          <>
+            <TouchableOpacity onPress={setDistruberComplete} style={styles.closeButton}>
+              <Icon.Close color={colors.darkText} size={normalize(16)} />
+            </TouchableOpacity>
+
+            {!!closestItem.picture && (
+              <Image
+                source={closestItem.picture}
+                borderRadius={normalize(8)}
+                aspectRatio={closestItem.picture.aspectRatio || { HEIGHT: 0.7, WIDTH: 1 }}
+                resizeMode={'cover'}
+              />
+            )}
+
+            <Wrapper style={styles.noPaddingBottom}>
+              <WrapperHorizontal>
+                {!!closestItem.category && (
+                  <BoldText center uppercase style={styles.categoryText}>
+                    {closestItem.category}
+                  </BoldText>
+                )}
+              </WrapperHorizontal>
+            </Wrapper>
+
+            <Wrapper style={styles.noPaddingTop}>
+              <WrapperHorizontal>
+                {!!closestItem.title && (
+                  <BoldText center large>
+                    {closestItem.title}
+                  </BoldText>
+                )}
+              </WrapperHorizontal>
+            </Wrapper>
+
+            <Wrapper style={styles.noPaddingTop}>
+              <WrapperHorizontal>
+                {!!closestItem.description && (
+                  <RegularText center>{closestItem.description}</RegularText>
+                )}
+              </WrapperHorizontal>
+            </Wrapper>
+
+            <Wrapper>
+              <WrapperHorizontal>
+                {!!closestItem.button && (
+                  <Button
+                    extraLarge
+                    containerStyle={styles.containerRadius}
+                    title={closestItem.button.title || 'Mehr'}
+                    onPress={() =>
+                      navigation.navigate(
+                        closestItem.button.navigationTo,
+                        closestItem.button.params
+                      )
+                    }
+                  />
+                )}
+              </WrapperHorizontal>
+            </Wrapper>
+          </>
+        )}
+      </View>
+    </Wrapper>
+  );
+};
+
+const styles = StyleSheet.create({
+  categoryText: {
```

Comment:
please adjust naming here as well, considering that "category" is the wrong naming

Reply:
Why don't you like `category`? Should it be `headline`?

Reply:
yes, because it is not a category but context free text